Modules/Forum/classes/class.ilForumPost.php

```diff
@@ -164,13 +164,6 @@ public function update()
 				)
 			);
 			
-			if($this->objThread->getFirstPostId() == $this->id)
```

Comment:
Can you please remove this from this PR and create another (small) one with this change and a corresponding description? Thx in advance.